db/migrate/20231220115621_update_organization_sequential_id_index.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateOrganizationSequentialIdIndex < ActiveRecord::Migration[7.0]
+  def change
+    remove_index :invoices, name: :unique_organization_sequential_id
```

Comment:
Rails/ReversibleMigration: remove_index(without column) is not reversible. (https://rails.rubystyle.guide#reversible-migration, https://api.rubyonrails.org/classes/ActiveRecord/Migration/CommandRecorder.html)